### ISSUE TITLE
Transfer from previous data

### ISSUE THREAD

**Issue by dnpfclrnt** [dnpfclrnt/fist_chj#6]
For learning rate and hyper-parameter tuning for XGBoost model, loading previous modeless data will help comparing the quality of result.
